covquestions-editor-app/src/components/QuestionnaireExecution.tsx

```diff
@@ -94,38 +120,76 @@ export const QuestionnaireExecution: React.FC<QuestionnaireExecutionProps> = ({
   return doRerender ? (
     <></>
   ) : (
-    <div className={classes.root}>
-      <Grid container item xs={12} className={`${classes.paddingRight} grid-row`} justify="flex-end">
-        <Button onClick={restartQuestionnaire} variant="contained" color="secondary">
-          Restart Questionnaire
-        </Button>
-      </Grid>
-      {isJsonInvalid ? (
-        <Grid item xs={12} className={`${classes.paddingRight} grid-row`}>
-          <Alert severity="warning">Cannot load questionnaire. JSON is invalid!</Alert>
-        </Grid>
-      ) : null}
-      <Grid item xs={12} className={`${classes.paddingRight} grid-row ${classes.execution}`}>
+    <Grid container direction="column" justify="space-between" className={`${classes.padding} overflow-pass-through`}>
+      <Grid item className={`${classes.execution}`}>
+        <Typography className={classes.internalStateHeadline}>Questionnaire Preview</Typography>
+        {isJsonInvalid ? <Alert severity="warning">Cannot load questionnaire. JSON is invalid!</Alert> : null}
         {result === undefined && currentQuestion ? (
-          <QuestionComponent currentQuestion={currentQuestion} handleNextClick={handleNextClick} />
+          <Paper className={classes.root}>
+            Hi
+            <Grid container direction="column" alignItems="stretch">
+              <Grid item xs={12}>
+                <QuestionFormComponent
+                  currentQuestion={currentQuestion}
+                  onChange={setCurrentValue}
+                  value={currentValue}
+                />
+              </Grid>
+              {currentQuestion.details ? (
+                <Grid item xs={12}>
+                  <Grid item xs={12}>
+                    <Typography>Hint:</Typography>
+                  </Grid>
+                  <Grid item xs={12}>
+                    <Typography>{currentQuestion.details}</Typography>
```

Comment:
- The details seem to contain HTML, like <li> or <b>. Should display this here? Would probably need a HTML sanitizer and dangerously set inner HTML. => maybe another time since it is just a preview.
- Should the details come after the answer option or rather between question and options? Not sure how it is in the CovApp => would do it the same way.

Reply:
yes and yes, but currently the question is set directly inside the Question Form Component and I did not want to go down there, as I already changed quite a bit